src/components/cw20-token/services/cw20-token.service.ts

```diff
@@ -265,6 +208,18 @@ export class Cw20TokenService {
     return tokenData?.current_price || 0;
   }
 
+  async getTokenMarket(
+    ctx: RequestContext,
+    request: Cw20TokenMarketParamsDto,
+  ): Promise<any> {
+    this.logger.log(ctx, `${this.getPriceById.name} was called!`);
+    const tokenData = await this.tokenMarketsRepository.find({
+      where: { contract_address: In(request.contractAddress) },
```

Comment:
Chỗ này k check data truyền lên liệu FE truyền sai định dạng lên thì có lỗi k a

Reply:
Nếu truyền null thì sẽ bị exception, nhưng trên FE đã default là rỗng, để a add thêm check cho an toàn